src/building/bootstrapping.md

```diff
@@ -135,31 +135,6 @@ bootstrapping the compiler.
 [intrinsics]: ../appendix/glossary.md#intrinsic
 [ocaml-compiler]: https://github.com/rust-lang/rust/tree/ef75860a0a72f79f97216f8aaa5b388d98da6480/src/boot
 
-## Contributing to bootstrap
```

Comment:
Maybe link to src/bootstrap/README somewhere? At the very beginning with a warning that it's internal documentation, maybe?

Reply:
Yeah, good idea